### PR TITLE
Matthew/roberta

### PR DESCRIPTION
**Pull request by matthewbayer**

### CODE REVIEW

File: finetune/base_models/gpt2/encoder.py [hunk 1]
```diff
@@ -51,6 +51,21 @@ class GPT2Encoder(BaseEncoder):
 
     def __init__(self, encoder_path=ENCODER_PATH, vocab_path=VOCAB_PATH):
         super().__init__(encoder_path=encoder_path, vocab_path=vocab_path)
+        self.freqs = {}
+        index = 0
+        with open("dict.txt", "r", encoding="utf-8") as freq_dict:
```
Comment (lhz1029) [inline finetune/base_models/gpt2/encoder.py]: I only see mention of dict.txt for bert and roberta. Is there a separate one for GPT-2? And it'll be in the gpt2 directory?

File: finetune/base_models/gpt2/encoder.py [hunk 2]
```diff
@@ -51,6 +51,21 @@ class GPT2Encoder(BaseEncoder):
...
+            lines = freq_dict.readlines()
+            for line in lines:
+                idx = line.rfind(" ")
+                if idx == -1:
+                    raise ValueError(
+                        "Incorrect dictionary format, expected '<token> <cnt>'"
+                    )
+                if "madeupword" in line[:idx]:
```
Comment (lhz1029) [inline finetune/base_models/gpt2/encoder.py]: Seems like this might have been for testing?

Comment (madisonmay) [reply finetune/base_models/gpt2/encoder.py]: Maybe... but for Facebook's testing purposes :P

File: finetune/base_models/gpt2/encoder.py (hunk 1 above)
Comment (madisonmay) [reply finetune/base_models/gpt2/encoder.py]: Correct.  Slightly different name but same idea.

Comment (matthewbayer) [reply finetune/base_models/gpt2/encoder.py]: Dict.txt is not needed for gpt2 - sorry, will remove

File: finetune/base_models/gpt2/encoder.py (hunk 2 above)
Comment (matthewbayer) [reply finetune/base_models/gpt2/encoder.py]: No facebook literally has that in their files

**Pull request merged**